### PR TITLE
Detect Flutter Web apps and embed inspector page for them 

### PR DESCRIPTION
**Pull request by elliette**
This is still just restricted to internal apps at the moment.

### CODE REVIEW

File: dwds/debug_extension/web/inspector_panel.html [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+            <button class="material-button" id="debuggingButton" type="button" disabled>Inspect app</button>
+        </div>
+        <div id="iframeContainer"></div>
+    </div>
+
+    <!-- Load script: -->
+    <script src="panel.js"></script>
+</body>
+
+</html>
```
Comment (annagrin) [inline dwds/debug_extension/web/inspector_panel.html:21]: Do we need a new line here?

**Review by annagrin — COMMENTED** [review]
Thanks Elliott! Is there a way to add tests for the new panel? I think we have extension tests somewhere.

**Review by annagrin — APPROVED** [review]
LGTM with a small comment

File: dwds/debug_extension/web/inspector_panel.html (hunk 1 above)
Comment (elliette) [reply dwds/debug_extension/web/inspector_panel.html:21]: I don't think so - it is automatically removed by the formatter when I save the file

**Comment by elliette** [comment]
> Thanks Elliott! Is there a way to add tests for the new panel? I think we have extension tests somewhere.

I have an open TODO to see if that's possible: https://github.com/dart-lang/webdev/blob/master/dwds/test/debug_extension_test.dart#L130-L132 WebDriver doesn't support interacting with Chrome DevTools, so with our current test infrastructure there is no way to test for it.

File: dwds/debug_extension/web/devtools.js [hunk 2]
```diff
@@ -1,46 +1,82 @@
...
+      });
+  }
+
+  function createPanelsIfFlutterApp() {
+    const checkFlutterAppInterval = setInterval(function() {
+      if (checkFlutterCount++ > 10) {
+        // If Flutter is not detected after 10 seconds, still
+        // create the Dart Debugger panel:
+        clearInterval(checkFlutterAppInterval);
+        chrome.devtools.panels.create(
```
Comment (annagrin) [inline dwds/debug_extension/web/devtools.js:58]: Looks like for flutter the debugger panel is not created until the inspector one is available. Does it make sense to show the debugger panel always, and add the inspector panel when we can?

**Review by annagrin — APPROVED** [review]
LGTM with a comment.

File: dwds/debug_extension/web/devtools.js (hunk 2 above)
Comment (elliette) [reply dwds/debug_extension/web/devtools.js]: Changed to that!

**Pull request merged**